Server/Server.Core/Server.Core.Static/Server.cpp

```diff
@@ -18,7 +18,12 @@ bool Server::onClientConnect(const std::shared_ptr<Connection>& client)
     return true;
 }
 
-void Server::onClientDisconnect(const std::shared_ptr<Connection>& client) { std::cout << "Removing client [" << client->getID() << "]\n"; }
+void Server::onClientDisconnect(const std::shared_ptr<Connection>& client)
+{
+    std::stringstream out;
+    out << "Removing client [" << client->getID() << "]";
```

Comment:
This is not the best way to create regular string...
https://quick-bench.com/q/IzYOZ2eR2Qu6Lu93iLRvuRTHqZ8